src/windows/windows.es6

```diff
@@ -401,9 +401,6 @@ export const init = function($parentObj) {
    /* wrap jquery ui dialog('destory') to fire an event */
    var original = $.fn.dialog;
    $.fn.dialog = function(cmd) {
-      if(!this.data('ui-dialog')) {
-         return this;
-      }
```

Comment:
@aminroosta Actually now that you pointed out it turns out that the problem is with that line itself. With that line removed everything seems to work fine.